Realm/RLMRealm.mm

```diff
@@ -447,6 +447,24 @@ + (instancetype)realmWithConfiguration:(RLMRealmConfiguration *)configuration
         }
     }
 
+    if (configuration.seedFilePath) {
+        DB::call_with_lock(configuration.config.path,
+                           [&configuration, &error](auto const&) {
+            if (![RLMRealm fileExistsForConfiguration:configuration]) {
+                @autoreleasepool {
+                    bool didCopySeed = false;
+                    NSError *copyError;
+                    didCopySeed = [[NSFileManager defaultManager] copyItemAtURL:configuration.seedFilePath
+                                                                          toURL:configuration.fileURL
+                                                                          error:&copyError];
+                    if (!didCopySeed) {
+                        RLMSetErrorOrThrow(copyError, error);
```

Comment:
Need to return nil after this, or we'll go on to try to open the file after failing to copy if `error` is non-nil.